Sources/Plasma/PubUtilLib/plAvatar/plArmatureEffects.h

```diff
@@ -78,8 +79,8 @@ class plArmatureEffectsMgr : public hsKeyedObject
 
     bool MsgReceive(plMessage* msg) override;
 
-    uint32_t GetNumEffects();
-    plArmatureEffect *GetEffect(uint32_t num);
+    size_t GetNumEffects();
+    plArmatureEffect *GetEffect(size_t num);
```

Comment:
```suggestion
    size_t GetNumEffects() const;
    plArmatureEffect *GetEffect(size_t num) const;
```

I also wouldn't stroke out if the definition were inlined :wink: